### DISCUSSION TITLE
Outline: Provider is unavailable

### DISCUSSION THREAD

**Discussion by qwertytam** [atom/atom#22596] [Q&A]
Hi,

Whenever I start Atom with a python (`*.py`) file open the Outline tab has the following message: `Provider is unavailable. Looks like a provider for this type of file is not available. Check if a relevant IDE language package is installed and has outline support, or try adding one from Atom's package registry (e.g.: atom-ide-javascript, atom-typescript, ide-python, ide-ruse, ide-css, ide-json).`

I have the following community packages installed which I think should meet the requirement in the error message:

- `atom-ide-base`
- `atom-ide-console`
- `atom-ide-debugger-python`
- `atom-ide-ui`
- `ide-python`

Any suggestions for diagnostics to see why Outline is not loading/working?

Windows 10, Python 3.8.5, Atom 1.57.0, all Atom packages up to date

**Comment by wajhk** [comment]
في اثنين، 14 يونيو، 2021 في 4:10 م، كتب qwertytam ***@***.***
